e2e/actors/base.js

```diff
@@ -212,7 +222,8 @@ module.exports = {
 
   async navigateToCaseDetails(caseId) {
     this.amOnPage(`${baseUrl}/cases/case-details/${caseId}`);
-    I.wait(0.5);
+    // I.wait(0.5);
+    I.waitForText('CCD ID');
```

Comment:
this is a generic function, called from other methods as well, changing this might break other methods, may be write a new one for your test to wait for ccd id?